docs/breaking-changes.md

```diff
@@ -1265,6 +1268,10 @@ not present, then the native module will fail to load on Windows, with an error
 message like `Cannot find module`. See the [native module
 guide](/docs/tutorial/using-native-node-modules.md) for more.
 
+### Removed: IA32 Linux support
+
+See - [Discontinuing support for 32-bit Linux](https://www.electronjs.org/blog/linux-32bit-support).
```

Comment:
```suggestion
Electron 18 will no longer run on 32-bit Linux systems. See [discontinuing support for 32-bit Linux](https://www.electronjs.org/blog/linux-32bit-support) for more information.
```

Sorry to nit, but could we add a very small descriptor here in addition to linking to the blog post (something similar to what you included in the PR comments)? Just thinking of devs scanning the breaking changes sheet when upgrading Electron, wanted to give them a bit more information 🙂